### PR TITLE
Handling stderr and stdout fo check_pods()

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
closes : #54 
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/77)

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
Added comments in issue.

File: deploy/gk-deploy [hunk 1]
```diff
@@ -138,16 +141,22 @@ assign() {
...
-    pods=$(${CLI} get pod --no-headers --show-all --selector="${1}")
-    debug "Checking status of pods matching '${1}':"
-    debug "$pods"
+    pods=$(${CLI} get pod --no-headers --show-all --selector="${1}" 2>&1)
+    if [[ "${prestatus}" != "${pods}" ]]; then
+      debug "Checking status of pods matching '${1}':"
+      debug "$pods"
+      prestatus="${pods}"
+    else
+      debug "dotit"
```
Comment (jarrpa) [inline deploy/gk-deploy]: Just send `debug -n .` here.

Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: Awesome. Maybe I thought too much like having it as a functionality ;P, I didn't realize we can do this.

File: deploy/gk-deploy [hunk 2]
```diff
@@ -152,10 +149,10 @@ check_pods() {
     pods=$(${CLI} get pod --no-headers --show-all --selector="${1}" 2>&1)
     if [[ "${prestatus}" != "${pods}" ]]; then
       debug "Checking status of pods matching '${1}':"
-      debug "$pods"
+      debug "-n" "$pods"
```
Comment (jarrpa) [inline deploy/gk-deploy]: No `-n` here, that would cause no newline to be printed after the debug output, which we want in this case.

**Comment by jarrpa** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/gluster/gluster-kubernetes/77#-:-KZ1w25RidgVWiNVcroz:bnfp4nl)*

**Pull request merged**